modInterface.py

```diff
@@ -109,7 +109,6 @@ def showExpiration(self):
         output = self.formatExpirationResult(data)
         
         self.txtDisplay.insert(END, output)
```

Comment:
Change this to " self.display(output)  ".
We want to call the display method so that it can make the changes to the text element, instead of this method making the change.

Reply:
suggested changes have been implemented